internal/backends/python/python.go

```diff
@@ -368,6 +368,12 @@ func makePythonPipBackend(python string) api.LanguageBackend {
 			span, ctx := tracer.StartSpanFromContext(ctx, "pip install")
 			defer span.Finish()
 
+			normalizedPkgs := make(map[api.PkgName]api.PkgName)
+			for name := range pkgs {
+				// Deleting this denormalizes the output of upm list, fixing Workspace bug, but breaks upm add --lang pip
```

Comment:
> fixing Workspace bug

Can we reference or describe the bug so this isn't scary to change later?

Reply:
I think that was a WIP comment, it doesn't read well. I'll fix it up, thanks

Reply:
Fleshed that out a bit. The underlying issue was Workspace was attempting to install `discord.py`, which we were normalizing and returning as `discord-py`, which would then (rightfully) confuse the workspace.

Hopefully the comment makes more sense.